src/main/java/com/bandwidth/iris/sdk/model/ImportTnOrder.java

```diff
@@ -67,6 +69,38 @@ public static OrderHistoryWrapper GetHistory( IrisClient client, String orderId
         return client.get( client.buildAccountModelUri( new String[] {IrisPath.IMPORT_TN_ORDERS, orderId, "history" } ), OrderHistoryWrapper.class);
     }
 
+    public static FileListResponse GetLoas( IrisClient client, String orderId ) throws Exception {
+        return client.get( client.buildAccountModelUri( new String[] {IrisPath.IMPORT_TN_ORDERS, orderId, IrisPath.LOAS } ), FileListResponse.class);
+    }
+
+    public static void PostLoasFile( IrisClient client, String orderId, File file, String contentType ) throws Exception {
+        client.postFile( client.buildAccountModelUri( new String[] {IrisPath.IMPORT_TN_ORDERS, orderId, IrisPath.LOAS } ), file, contentType);
+    }
+
+    public static byte[] GeLoasFile( IrisClient client, String orderId, String loasId ) throws Exception {
+        return client.getFile( client.buildAccountModelUri( new String[] {IrisPath.IMPORT_TN_ORDERS, orderId, IrisPath.LOAS, loasId } ) );
+    }
+
+    public static void PutLoasFile( IrisClient client, String orderId, String fileid, File file, String contentType ) throws Exception {
```

Comment:
Similar for line 76, `ReplaceLoasFile` would be a preferred name if it would make sense